jsapp/js/components/permissions/userPermissionRow.es6

```diff
@@ -120,8 +120,13 @@ class UserPermissionRow extends React.Component {
           >
             {permName}
 
-            {permUsers.length > 0 &&
+            { // KEY_CODES.NBSP doesn't work for me in Chrome or FF, but it's
+              // okay because (for now) we're not including this username list
+              // anyway
+              /*
+              permUsers.length > 0 &&
               String.fromCharCode(KEY_CODES.NBSP) + '(' + permUsers.join(', ') + ')'
+              */
```

Comment:
Ouch! Maybe we should remove this code and `KEY_CODES.NBSP` instead of commenting out?

Reply:
As discussed, I tried showing a maximum of three usernames, with (`and ## more`) as a placeholder when the number of assigned users exceeds that. I'm not sure if I like the result or not:
![image](https://user-images.githubusercontent.com/2085013/65019029-6638d380-d8f8-11e9-85d6-daf2b3524acb.png)
I'll push the code anyway.

Reply:
It's not perfect, I agree, but I think it's more than OK for initial version of row level perms :) Let's merge this PR?